README.md

```diff
@@ -45,8 +54,18 @@ var Panda = function(name, age) {
 }
 ```
 Your Answer:
-```js
-// your code here
+```
+var Panda = function(name, age) {
+  this.name = name;
+  this.age = age;
+  this.num_bamboo_eaten = 0;
```

Comment:
Almost, but we would want to put this method `eat_bamboo` on the prototype instead of the constructor: 
```
Panda.prototype.eat_bamboo = function(){
  this.num_bamboo_eaten += 1;
}
```